lightning_talk/models/post.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+from django.conf import settings
+from .upvote import Upvote
+
+
+class Post(models.Model):
+    title = models.CharField(max_length=40)
+    description = models.CharField(max_length=200)
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL, on_delete=models.SET_NULL, null=True)
+    upvotes = models.ManyToManyField(Upvote, blank=True)
```

Comment:
The way to represent one-to-many relations in Django is to use ForeignKey in the "many" end and Django automatically creates reverse-mapping.

To do this:
- Create `post` field on Upvote model that is a ForeignKey
- To get all upvotes for a post call `post.upvote_set.all()`. The `upvote_set` is a "magical" variable name that is automatically generated to get all upvotes that reference this post.